jck/jtrunner/JavaTestRunner.java

```diff
@@ -62,12 +62,9 @@ public class JavaTestRunner {
 	private static String nativesLoc;
 	private static String jckConfigLoc;
 	private static String initialJtxFullPath;
-	private static String jtxFullPath;
-	private static String jtxDevFullPath;
-	private static String customJtx;
+	private static String defaultJtxFullPath;
 	private static String kflFullPath;
-	private static String testFlagJtxFullPath;
-	private static String[] testFlagList;
+	private static String customJtx;
```

Comment:
should that be customJtxPath?  Hmm, the logic below suggests this name is OK so I tihnk you can ignore my comment here